plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerTypeMapping.java

```diff
@@ -580,6 +581,91 @@ public void testSqlServerDatetime2()
                 .execute(getQueryRunner(), sqlServerCreateAndInsert("test_sqlserver_timestamp"));
     }
 
+    @Test(dataProvider = "testTimestampDataProvider")
+    public void testSqlServerDatetimeOffset(ZoneId sessionZone)
+    {
+        Session session = Session.builder(getSession())
+                .setTimeZoneKey(TimeZoneKey.getTimeZoneKey(sessionZone.getId()))
+                .build();
+
+        SqlDataTypeTest.create()
+                // With +00:00 time zone
+                .addRoundTrip("DATETIMEOFFSET", "'1970-01-01 00:00:00'", createTimestampWithTimeZoneType(7), "TIMESTAMP '1970-01-01 00:00:00.0000000+00:00'")
```

Comment:
Is default UTC offset or the server timezone? Can you verify once? If default is server timezone then let's make the tests use the +00:00 offset explicitly.

Reply:
It will default to `+00:00` regardless of server timezone.